packages/geolocation/src/drivers/mongodb/connection.js

```diff
@@ -0,0 +1,30 @@
+const { MongoClient } = require('mongodb');
+
+const config = require('../../../config/mongodb');

```

Comment:
I create a new way to make connection mongoose

### Steps to be able to use it

1.  update your branch
2. run `yarn bootstarp` in root directory

### Steps to implement it

1. import MongoDriver in your file (notification-system/src/drivers/mongo/connection.js) `const { MongoDriver } = require('@booking-services/shared');`
2. call the function giving a prefix like a parameter 
example about how my team implement it.

```
const { MongoDriver } = require('@booking-services/shared');
const { prefix } = require('../../utils/constants')

module.exports = MongoDriver.createConnection(prefix);
```

**Note:** createConnection return an instance of mongoose so you can use a variable something like this
`const connection = MongoDriver.createConnection(prefix);
module.exports =  connection;
`